mobile/native/src/api.rs

```diff
@@ -327,6 +329,17 @@ pub fn order_matching_fee(quantity: f32, price: f32) -> SyncReturn<u64> {
     SyncReturn(order_matching_fee)
 }
 
+/// Calculates the max quantity the user is able to trade considering the trader and the coordinator
+/// balances and constraints. Note, this is not an exact maximum, but a very close approximation.
+pub fn calculate_max_quantity(price: f32, trader_leverage: f32) -> SyncReturn<u64> {
+    let price = Decimal::from_f32(price).expect("price to fit in Decimal");
+
+    let max_quantity = max_quantity(price, trader_leverage).unwrap_or(Decimal::ZERO);
```

Comment:
Should we print the error if we get one?

Reply:
I don't think its very useful. If this errors we would see that message already in other places.

Reply:
Why not error here?